app/src/main/res/layout/activity_main.xml

```diff
@@ -9,8 +9,70 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     tools:context="com.example.molliestephenson.tictactoe.MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!" />
+    <Button
+        android:layout_width="100dp"
```

Comment:
Can `android:layout_width="..."` and everything else that all buttons have in common be extracted into something like a style (_that is what XAML call is in the .Net world_)